python/unityagents/environment.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, file_name, worker_id=0,
         atexit.register(self.close)
         self.port = base_port + worker_id
         self._buffer_size = 12000
-        self._python_api = "API-2"
+        self._python_api = "API-3"
```

Comment:
Can we call the variable name `_version_`  in both C# and Python, i.e. 

`self._version_ = "API-3"` or even `self._version_ = "0.3"` and `const string _version_ = "API-3"`

Reply:
Done.